docs/install.rst

```diff
@@ -393,6 +393,22 @@ packages that may be useful for radio astronomy).
    <https://github.com/astropy/astropy/issues?q=+label%3ACASA-Installation+>`_
    and if you do not encounter your issue there, please post a new one.
 
+
+Installing pre-built Development Versions of ``astropy``
+--------------------------------------------------------
+
+Most nights a development snapshot of ``astropy`` will be compiled.
+This is useful if you want to test against a development version of astropy but
+do not want to have to build it yourself.
+
+Installing these "nighlies" of ``astropy`` can be achieved by using pip::
+
+  $ pip install --extra-index-url=https://pkgs.dev.azure.com/astropy-project/astropy/_packaging/nightly/pypi/simple/ --pre astropy
+
+The extra index URL tells pip to check the pip index on Azure where the
+nightlies are built, and the ``--pre`` command tells pip to install pre-release
+versions (in this case ``.dev`` releases).
```

Comment:
So, when we push out RC, will `--pre` favor the RC over dev, or not?

Reply:
yes, those version numbers are higher.

Reply:
But you don't push RC to Azure, right? So maybe there's no ambiguity there?